README.md

```diff
@@ -19,8 +19,9 @@ This is a monorepo containing all officially maintained Cisco Webex JS SDK modul
 - [The Cisco Webex JS SDK](#the-cisco-webex-js-sdk)
   - [Install](#install)
   - [Usage](#usage)
-      - [_A note on browser usage_](#a-note-on-browser-usage)
-      - [_Still using `webex/env` or `ciscospark/env`?_](#still-using-webexenv-or-ciscosparkenv)
+    - [Browser Usage](#browser-usage)
+      - [Browser Usage Example](#browser-usage-example)
+    - [Node Usage](#node-usage)
```

Comment:
Would be cool to drop a small node usage example.

Reply:
We've got links below it to the quick usage guide in documentation for now.